### DISCUSSION TITLE
Is it possible to switch GA tags? Or have multiple GA tags?

### DISCUSSION THREAD

**Discussion by arnotixe** [capawesome-team/capacitor-firebase#606] [Ideas]
Use case: I have a POS/checkout app (POS as a service). Analytics work well now BUT all the analytics come in on my central analytics account (of course).

Would it be possible to switch the GA tag to the different user's own one, after they log in? Would make more sense that each customer collect their own sales data.

In my head I'm thinking that I should actually run a kind of parallel setup: the purchasing and app functionality analytics sent to different places. (app functionality to me, purchasing to the customer's).

Just not sure if that's possible (or smart)

**Comment by robingenz** [comment]
Does the Firebase Android/iOS SDK offer such a feature? If yes, we could implement it. I'm not that familiar with GA.
